test/pytest/pseudo.py

```diff
@@ -1,15 +1,16 @@
 import os
-import json
 import sys
 
+from cockpit._vendor.ferny import interaction_client
+
 pw = os.environ.get('PSEUDO_PASSWORD')
 if pw:
-    challenge = '\n{"command":"authorize", "challenge":"plain1:", "cookie":"dontcare", "prompt": "can haz pw?"}\n'
-    sys.stdout.write(f'{len(challenge)}\n{challenge}')
-    sys.stdout.flush()
+    reader, writer = os.pipe()
+    interaction_client.interact(2, writer, ['-', 'can haz pw?'], {})
```

Comment:
> interact() is expecting to get sys.argv as that argument, so '-' is a stand-in for where the program name would be.

Do you mind adding a comment what the `-` does?

Reply:
OK